
import attr_Error

print(dir(attr_Error))
print(help(attr_Error))
